views/administration/forms/form-terminate-school-year.hbs

```diff
@@ -1,2 +1,2 @@
-<p>Das Schuljahr {{school.currentYear.name}} wird beendet und die Transferphase zum neuen Schuljahr beginnt. <strong>Dieser Schritt kann nicht rückgängig gemacht werden.</strong> Es können in dieser Zeit keine Klassen und Nutzer angelegt oder geändert werden.</p>
-<p>Die Synchronisierung von LDAP-Stammdaten wird während der Transferphase unterbrochen. Somit werden Nutzerdaten und Klassen (sofern verwendet) nicht mehr über LDAP aktualisiert. Der Login mit bereits synchronisierten Login-Daten ist weiterhin möglich. Passwortänderungen im LDAP während des Logins werden ebenfalls berücksichtigt.</p>
+<p>{{$t "administration.form_TerminateSchoolYear.text.theSchoolYear" }} {{school.currentYear.name}} {{$t "administration.form_TerminateSchoolYear.text.isEndedAndTheTransferPhase" }} <strong>{{$t "administration.form_TerminateSchoolYear.text.thisStepCannotBeUndone" }}</strong> {{$t "administration.form_TerminateSchoolYear.text.noClassesAndUsersCanBeCreated" }}</p>
```

Comment:
better to have the full paragraph in one key

Reply:
done ;)